cms/jinja2/templates/components/streamfield/table_block.html

```diff
@@ -1,2 +1,13 @@
-{% from "components/table/_macro.njk" import onsTable %}
-{{ onsTable(options) }}
+{% from "component_overrides/table/_macro.njk" import onsTable %}
+<figure id="table-{{ block_id }}">
+    {% if title %}<h3 class="heading-4">{{ title }}</h3>{% endif %}
+
+    {{ onsTable(options) }}
+
+    {%- if source or footnotes %}
+        <footer>
+            {% if source %}<p>{{ _("Source") }}: {{ source }}</p>{% endif %}
+            {% if footnotes %}{{ footnotes|richtext }}{% endif %}
+        </footer>
+    {% endif %}
```

Comment:
Couldn't the `title` and `footnotes` be used in the custom macro for the table instead? 

I'm assuming the long term plan is to include this new version of the table into the DS. This way it'd reduce the need for modification of this file in the future.

Reply:
Fair point, but that is a Design System question. 
I wanted to make minimum modifications as I don't know the direction in which DS will take it. If it gets incorporated, then will happily use it